initialize_vault_store.py

```diff
@@ -0,0 +1,36 @@
+import json
+import os
+from authx.auth import set_service_store_secret, add_provider_to_opa
+import sys
+
+results = []
```

Comment:
results is set and appended to, but never seems to be read. Was this just for debugging?

Reply:
added a commented-out print so that's clearer.